app/app.go

```diff
@@ -190,10 +190,10 @@ type App struct {
 	BasicManager  module.BasicManager
 	ModuleManager *module.Manager
 	configurator  module.Configurator
-	// timeoutCommit is used to override the default timeoutCommit. This is
+	// blockTime is used to override the default TimeoutHeightDelay. This is
```

Comment:
Is TimeoutHeightDelay the same as timeout commit in function in that it is the time following from the commit with which to wait before starting the next height?

Reply:
https://github.com/celestiaorg/celestia-app/pull/5639/commits/b8e2364290b10e071a2070063c6d870a63eb5bf9